rpc/src/v1/types/confirmations.rs

```diff
@@ -38,8 +39,25 @@ impl From<helpers::ConfirmationRequest> for ConfirmationRequest {
 	}
 }
 
+impl fmt::Display for ConfirmationRequest {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+		write!(f, "#{}: {}", self.id, self.payload)
+	}
+}
+
+impl fmt::Display for ConfirmationPayload {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+		match *self {
+			ConfirmationPayload::Transaction(ref transaction)
+				=> write!(f, "{}", transaction),
+			ConfirmationPayload::Sign(_) => write!(f, "TODO: data"),
```

Comment:
Should be still Todos? Will it be covered in next PR? I think just displaying a `From: ` and type of operation would be sufficient here.